src/client/service-worker/events/utility.js

```diff
@@ -127,7 +127,12 @@ export function fetchCacheResource(event, meta) {
   return match(event.request.clone(), { cacheName: meta.cacheName })
     .then(
       (cachedResponse) => cachedResponse
-      || fetch(event.request.clone()).then(setCacheResource(event, meta))
+      || fetch(event.request.clone()).then((response) => {
+        if (response.status === 200) {
```

Comment:
Wondering if we have to worry about other success status codes like what happens with 304 for example?🤔

Reply:
It would be cached initially right? 🤔